tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -118,8 +118,8 @@ namespace Datadog.Trace
     }
     public static class Tags
     {
-        public const string DbName = "db.name";
-        public const string DbType = "db.type";
+        public const string DbName = "db.instance";
```

Comment:
Is it okay to change these public constants? Or should we introduce new constants like `string DbInstance = "db.instance"`?

Reply:
My guess was that if someone uses it, they use it to match our tags, and thus it was less breaking than adding another one (which BTW I honestly think we shouldn't do, I don't see the value in having this in the public api).

Reply:
Sounds good. Perhaps we can remove them in the next major version